x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/service.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ExceptionListItemSchema,
+  FoundExceptionListItemSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import { ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID } from '@kbn/securitysolution-list-constants';
+import { HttpStart } from 'kibana/public';
+import { EXCEPTION_LIST_ITEM_URL, EXCEPTION_LIST_URL } from '../event_filters/constants';
+import { HOST_ISOLATION_EXCEPTIONS_LIST } from './constants';
+
+async function createHostIsolationExceptionList(http: HttpStart) {
```

Comment:
fwiw, you can use ES6, fat arrow notations with async and anything else. I don't know the style mixtures happening everywhere between these or if there's a good reason why you want to mix between styles, but just pointing it out. Not changes asked, just an observation.

I would do what you are doing below and add explicit return types to these functions as that helps typescript and the next programmer to not accidentally mess up the return types by introducing a new code path or something odd.

For example:

```ts
async function createHostIsolationExceptionList(http: HttpStart): Promise<void> {
```

Reply:
>fwiw, you can use ES6, fat arrow notations with async and anything else.

I know this. I like to use named functions for named functions and arrow functions for anonymous functions.

>I would do what you are doing below and add explicit return types to these functions as that helps typescript and the next programmer to not accidentally mess up the return types by introducing a new code path or something odd.

Good idea. I was wondering why we don't have a eslint rule for this. I'll change it.